src/Tools/ExternalAccess/VisualDiagnostics/Contracts/IHotReloadDiagnosticManager.cs

```diff
@@ -0,0 +1,31 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Generic;
+using System.Collections.Immutable;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.VisualDiagnostics.Contracts;
```

Comment:
this is an odd name.  What is 'VisualDiagnostics'?

Reply:
VisualDiagnostics is part of debugging / hot reload / XAML hot reload. The whole PR is to enable hot reload error reporting :)